2022/day9/Program.fs

```diff
@@ -0,0 +1,91 @@
+﻿// https://adventofcode.com/2022/day/9
+open System.IO
+
+type Pos = struct
```

Comment:
This is perfectly fine, but a bit of uncommon syntax, you'll usually see something like this:
```
[<Struct>]
type Position = {
  x : int
  y : int
}
```
Both create a record If you want to add a method to it for example for chaining adding points:
```
type Position = {
  x : int
  y : int
}
  with member this.Add(p : Position) = {x = this.x + p.x; y = this.y + p.y}

let p1 = {x = 1; y = -1}
let p2 = {x = -1; y = 1}
let p3 = p1.Add(p2)
printfn "%A" p3 // { x = 0; y = 0 }
```